CHANGES.md

```diff
@@ -55,6 +55,7 @@
 - Always warn about uncommitted changes at the start of `dune-release
   distrib` (#325, @lehy).  Otherwise uncommitted changes to
   dune-project would be silently ignored by `dune-release distrib`.
+- Use the <rev>^0 syntax instead of <rev>^{commit} for revision dereference, as cygwin expands curly brackets into a matching expression, resulting in an unknown revision (#329, @gpetiot)
```

Comment:
How about something a bit less detailed about the implementation and more oriented towards what it fixes:
"Fixes a bug under cygwin where dune-release was unable to find the commit hash corresponding to the release tag"

What do you think?

Reply:
It's better indeed